README.md

```diff
@@ -344,8 +344,35 @@ For a comparison of some of these, see this
 
 ## Storage Backends
 
-In order to not download or scan any previously scanned sources again, the _scanner_ can use a storage backend to store
-scan results for later reuse.
+In order to not download or scan any previously scanned sources again or to reuse scan results generated via other
+services, the _scanner_ can be configured to use so-called storage backends. Before processing a package, it checks
+whether compatible scan results are already available in one of the storages declared; if this is the case, they
+are downloaded and reused. Otherwise, the package is scanned now. Afterwards, the new results can be put into a
+storage for later reuse.
+
+It is possible to configure multiple storages to read scan results from and to write new results to independently.
+In this configuration, the order of the storages is relevant, too. This allows a fine-grained control over the
+sources, from which existing scan results are loaded. For instance, you can specify that the scanner checks first
+whether results for a specific package are available in a local storage on the file system. If this is not the case,
+it can look up the package in a Postgres database. If this does not yield any results either, a service like
+[ClearlyDefined](https://clearlydefined.io) can be queried. Only if all of these steps fail, the scanner has to
+actually process the package.
+
+The configuration of storage backends is located in the
+[ORT configuration file](./model/src/test/assets/reference.conf). The file has a section named _storages_ that lists
+all the storage backends available and assigns them a name. Each storage backend is of a specific type and needs to be
+configured with type-specific properties. The different types of storage backends supported by ORT are described
+below.
+
+After the declaration of the storage backends available, the configuration file has to specify, which ones of them the
+scanner should use for looking up existing scan results or to store new results. This is done in two list properties
+named _storageReaders_ and _storageWriters_. The lists reference the names of the storage backends declared in the
+_storages_ section. As mentioned before, the order of these lists is relevant to define a priority for look-up
```

Comment:
I think it might be fine to drop the "As mentioned before" sentence.

Reply:
Dropped and slightly reworded to be better in-line with changes before.